reading_json.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import json
+import sys
+
+
+# FINISH TESTING & DEBUGGING
+#
+#
+
+def main():
+
+    # getting cmd_line_args in format:
+    # "python3 reading_json.py file.json img_dir output_dir"
+    cmd_line_args = sys.argv
+    print(cmd_line_args)
+    num_args = len(cmd_line_args)
+
+    # check for correct num of args (expecting 4 args: reading_json.py, file.json, img dir, output_dir)
+    if (num_args != 4):
+        # caller has wrong num of args
+        print("ERR: Arguments entered incorrectly. Please call with the command: 'python3 reading_json.py file.json img_dir output_dir' ")
+        return False
+
+    print("copying images from " + cmd_line_args[2])
+
+    # read input JSON file
+    j_file = (open(cmd_line_args[2])).read()
+
+    # parse JSON into a dictionary:
+    json_dict = json.loads(j_file)
+
+    # get images
+
+    imgs = []
+    for panel in json_dict["damaged"]:
+        imgs += [ panel['filename'] ]
+
+    # source file and destination file locations
+    src = cmd_line_args[3]
+    dest = cmd_line_args[4]
+
+    # getting image file locations and moving them to a destination folder
+    src_files = os.listdir(src)
+    for file_name in imgs:
+        full_file_name = os.path.join(src, file_name)
+        if (os.path.isfile(full_file_name)):
+            shutil.copy(full_file_name, dest)
+
+    return True
+
+
+main()
```

Comment:
You see that red block symbol GitHub put there? That means theres no newline at the end of the file. Usually you want the last line of your file to be blank (ask me if you want to know why).

Reply:
Hahaha I'm gonna take the bait ask why. I've never heard of this newline thing before.

Reply:
Its because of old standards /shrug

https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Ahhh because some programs have trouble processing files that don't end in a newline. Take a look at this. Its an interesting read! https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline